### ISSUE TITLE
Error: invalid argument

### ISSUE THREAD

**Issue by aaduo01** [JeanLucPons/Kangaroo#13]
Hi! Im error: invalid argument 
Windows server 2019
Windows 10 ok.
What problem? 
Bitcrack and vs-vanity-hybrid run without error, but bsgs and kangaroo no.

**Comment by djarumlights** [comment]
What're you entering to try and run it?

**Comment by aaduo01** [comment]
> What're you entering to try and run it?

cmd Kangaroo.exe
Windows 10 work no problem, but windows server no

**Comment by djarumlights** [comment]
Hm, interesting. I am not 100% certain why it would work on 10 but not Server 2019 unless there's some subtle differences in the background. Will have to wait on dev to respond.

**Comment by aaduo01** [comment]
> Hm, interesting. I am not 100% certain why it would work on 10 but not Server 2019 unless there's some subtle differences in the background. Will have to wait on dev to respond.

Ye, interesting.

**Comment by CatfishCrypt** [comment]
Have you tried running with a batch file on Win Server 19? Should be no difference in the cmd function. I have never used cmd Kangaroo.exe...mine just starts with Kangaroo.exe

**Comment by aaduo01** [comment]
run debug this output

'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\ntdll.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\kernel32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\KernelBase.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\advapi32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\msvcrt.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\sechost.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\rpcrt4.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\shell32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\cfgmgr32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbase.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\SHCore.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\combase.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\bcryptprimitives.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\windows.storage.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\msvcp_win.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\profapi.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\powrprof.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\shlwapi.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\gdi32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\gdi32full.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\user32.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\win32u.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\kernel.appcore.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\cryptsp.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\msvcp140d.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140d.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbased.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140_1d.dll'. Symbols loaded.
'Kangaroo.exe' (Win32): Loaded 'C:\Windows\System32\imm32.dll'. Symbols loaded.
The thread 0x7ac has exited with code 0 (0x0).
The thread 0x98 has exited with code -1 (0xffffffff).
The thread 0x1a84 has exited with code -1 (0xffffffff).
The program '[7432] Kangaroo.exe' has exited with code -1 (0xffffffff).

**Comment by RB61** [comment]
try running as administrator to see if it works?

**Comment by aaduo01** [comment]
> try running as administrator to see if it works?

Runing admin

**Comment by aaduo01** [comment]
Problem this administrator account not run, and admin cmd same problem windows 10 and server.
How to fix?

**Comment by aaduo01** [comment]
Kangaroo v1.3
Error: Cannot open  Invalid argument

**Comment by kpot87** [comment]
@aaduo01 Antivirus? Block malware programs? Do you have?

**Comment by aaduo01** [comment]
No.

**Comment by DvR4** [comment]
> Hi! Im error: invalid argument
> Windows server 2019
> Windows 10 ok.
> What problem?
> Bitcrack and vs-vanity-hybrid run without error, but bsgs and kangaroo no.

What parameters do you run with? This error means that you specified something incorrectly.

**Comment by JeanLucPons** [comment]
Error report with invalid input args has been improved in the 1.4
Please open a new issue if you find something wrong
Thanks
